src/styles.scss

```diff
@@ -1 +1,17 @@
-/* You can add global styles to this file, and also import other style files */
+$text: #ffffff;
+
+//I don't know why application create a white gap at the top and at the bottom
```

Comment:
There are more things in heaven and earth, Horatio, than are dreamt of in your philosophy.